pkg/encryption/path.go

```diff
@@ -159,3 +155,49 @@ func decryptPathComponent(comp string, cipher storj.Cipher, key *storj.Key) (str
 
 	return string(decrypted), nil
 }
+
+func encodeSegment(segment []byte) []byte {
+	if len(segment) == 0 {
+		return segment
+	}
+
+	// escape 0x00 to `{0x00, 0x01}`
+	// escape `/` to `{0x00, 0x02}`
```

Comment:
Can we use constants instead of magic numbers?

Reply:
sure